improve layout of DlgTrackInfo / Track Properties

ronso0 opened the pull request:
Includes some tweaks for the file info tab
* better alignment in general
* remove pointless section titles
* bold file propertie vlaues

and makes the BPM tab much clearer
* BPM convertion buttons have more space to expand with long translation strings
* "Clear beatgrid ..." button is now separate

![Screenshot_2020-04-27_16-54-03](https://user-images.githubusercontent.com/5934199/80386525-c8eb4800-88a7-11ea-8b3c-fb4d6cd2e94d.png)



master
![Screenshot_2020-04-22_19-58-51](https://user-images.githubusercontent.com/5934199/80016733-cf5a7800-84d3-11ea-9fe7-b827bdcd7799.png)

ronso0 commented:
As you can see in the screenshot, I tried to make use of the Date Added filed which was in te .ui file (still disabled for now).
If we decide to show this, as well, I need some help with properly formatting the date/time, preferably like the native format in the Date Added column (`dd-mm-yy hh:mm` for me) without weekday.

uklotzde commented:
BaseTrackTableModel contains the formatting code for dates:

```
} else if (column == fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_DATETIMEADDED)) {
        QDateTime gmtDate = rawValue.toDateTime();
        gmtDate.setTimeSpec(Qt::UTC);
        return gmtDate.toLocalTime();
}
```

It is already duplicated right there, we should better extract some utility functions for this purpose.

Holzhaus commented:
Can we make the BPM buttons + Tap expand to the whole width?

uklotzde commented:
> BaseTrackTableModel contains the formatting code for dates:
> 
> ```
> } else if (column == fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_DATETIMEADDED)) {
>         QDateTime gmtDate = rawValue.toDateTime();
>         gmtDate.setTimeSpec(Qt::UTC);
>         return gmtDate.toLocalTime();
> }
> ```
> 
> It is already duplicated right there, we should better extract some utility functions for this purpose.

Ups, not really. toLocalTime() still returns a QDateTime. Nevertheless this code is duplicated. I will check for a solution.

ronso0 commented:
When #2702 is merged I'll add the date here, and rebase.

Holzhaus commented:
Converting this to draft since you said you were going to rebase this.

ronso0 committed:
improve DlgTrackInfo layout, add 'Date added'

Includes some tweaks for the file info tab and makes the BPM tab
much clearer. "Clear beatgrid ..." button is now separate.

ronso0 commented:
Added 'Date added' returned by util/datetime.h, rebased.
Ready?

Holzhaus reviewed (approved):
LGTM, thank you!
